tasks/electron.js

```diff
@@ -73,7 +73,8 @@ function _processZipEntry(zip, dest, entry) {
   }
 }
 
-function _zipExtract(zipfile, dest = './') {
+function _zipExtract(zipfile, dest) {
+  dest = (typeof dest !== 'undefined') ? dest : './';
```

Comment:
Odd, this shouldn't be a problem at all. If I remember correctly Node 6 does support this out of the box, plus babel is used for the gulp tasks. I think something is wrong with your local setup rather then having to modify code.